app/server/appsmith-plugins/mongoPlugin/src/main/java/com/external/plugins/MongoPlugin.java

```diff
@@ -338,6 +338,21 @@ public void datasourceDestroy(MongoClient mongoClient) {
 
             }
 
+            boolean usingSrvUrl = endpoints
+                    .stream()
+                    .anyMatch(endPoint -> endPoint.getHost().contains("mongodb+srv"));
+
+            if(Boolean.TRUE.equals(usingSrvUrl)) {
```

Comment:
This form of checking isn't needed here since `usingSrvUrl` is of type `boolean`. It's only needed if the variable is of type `Boolean` (since on`boolean` variables cannot be set to `null`, but `Boolean` variables *can* be set to `null`).

```suggestion
            if(usingSrvUrl) {
```